man/systemd-cryptsetup.xml

```diff
@@ -3,31 +3,31 @@
 <!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
   "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
 <!-- SPDX-License-Identifier: LGPL-2.1-or-later -->
-<refentry id="systemd-cryptsetup@.service" conditional='HAVE_LIBCRYPTSETUP'>
+<refentry id="systemd-cryptsetup" conditional='HAVE_LIBCRYPTSETUP'>
 
   <refentryinfo>
-    <title>systemd-cryptsetup@.service</title>
+    <title>systemd-cryptsetup</title>
     <productname>systemd</productname>
   </refentryinfo>
 
   <refmeta>
-    <refentrytitle>systemd-cryptsetup@.service</refentrytitle>
+    <refentrytitle>systemd-cryptsetup</refentrytitle>
     <manvolnum>8</manvolnum>
   </refmeta>
 
   <refnamediv>
+    <refname>systemd-cryptsetup</refname>
     <refname>systemd-cryptsetup@.service</refname>
     <!-- <refname>system-systemd\x2dcryptsetup.slice</refname> — this causes meson to go haywire because it
          thinks this is a (windows) path. Let's just not create the alias for this name, and only include it
          in the synopsis. -->
-    <refname>systemd-cryptsetup</refname>
     <refpurpose>Full disk decryption logic</refpurpose>
   </refnamediv>
 
   <refsynopsisdiv>
+    <para><filename>systemd-cryptsetup</filename></para>
```

Comment:
This looks too superficial to me. Currently the man page only describes what `systemd-cryptsetup@.service` does, and doesn't cover the arguments/usage of `systemd-cryptsetup` itself at all.